lib/cdo/yaml.rb

```diff
@@ -27,6 +27,11 @@ def load_file(path)
       nil
     end
 
+    # Ensure YAML.safe_load allows aliases with Psych 4.
+    def safe_load(yaml, *args, **kwargs)
+      super(yaml, *args, **kwargs.merge(aliases: true))
```

Comment:
I think we _really_ don't want to do this; YAML aliases are an explicitly unsafe feature, which we should only allow when we're confident we have full control over the content being rendered. Burying the flag inside an override like this feels like a recipe for vulnerability